src/Auth0.OidcClient.Android/Resources/AboutResources.txt

```diff
@@ -1,44 +0,0 @@
-Images, layout descriptions, binary blobs and string dictionaries can be included 
```

Comment:
Why was this removed?

Reply:
Was just a boilerplate text file - added nothing to the lib.